rllib/algorithms/ppo/torch/ppo_torch_rl_module.py

```diff
@@ -1,20 +1,27 @@
-from dataclasses import dataclass, field
```

Comment:
A lot of stuff here, are implementation details that are heavily subject to change depending on model catalog design in action.